rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/TemplateUtils.java

```diff
@@ -14,10 +14,8 @@
 import retrofit.client.Response;
 
 import javax.xml.bind.DatatypeConverter;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
+import java.io.*;
```

Comment:
I don't think we want `*` here. We should be explicit about out imports.

Reply:
fixed